test/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepositoryTest.java

```diff
@@ -0,0 +1,313 @@
+        final int port = BASE_PORT + RANDOM.nextInt(1000);
+        final String jdbcUrl =  "jdbc:h2:./target/h2/metrics" +
+                RANDOM.nextInt(1000) +
+                ":portal;AUTO_SERVER=TRUE;AUTO_SERVER_PORT=" +
+                port +
+                ";MODE=PostgreSQL;INIT=create schema if not exists portal;DB_CLOSE_DELAY=-1";
+        return new FakeApplication(
+                new java.io.File("."),
+                Helpers.class.getClassLoader(),
+                ImmutableMap.of("db.metrics_portal_ddl.url", jdbcUrl, "db.default.url", jdbcUrl),
+                new ArrayList<String>(),
+                null);
+    }
+
+    @Before
+    public void setup() {
+        alertRepo.open();
+    }
+
+    @After
+    public void teardown() {
+        alertRepo.close();
+    }
+
+    @Test
+    public void testGetForInvalidId() {
+        Assert.assertFalse(alertRepo.get(UUID.randomUUID()).isPresent());
+    }
+
+    @Test
+    public void testGetForValidId() throws JsonProcessingException {
+        final UUID uuid = UUID.randomUUID();
+        Assert.assertFalse(alertRepo.get(uuid).isPresent());
+        final Transaction transaction = Ebean.beginTransaction();
+        try {
+            final models.ebean.Alert ebeanAlert = new models.ebean.Alert();
+            ebeanAlert.setNagiosExtension(TEST_NAGIOS_EXTENSIONS);
+            ebeanAlert.setName(TEST_NAME);
+            ebeanAlert.setOperator(TEST_OPERATOR);
+            ebeanAlert.setPeriod(TEST_PERIOD_IN_SECONDS);
+            ebeanAlert.setStatistic(TEST_STATISTIC);
+            ebeanAlert.setQuantityValue(TEST_QUANTITY_VALUE);
+            ebeanAlert.setQuantityUnit(TEST_QUANTITY_UNIT);
+            ebeanAlert.setCluster(TEST_CLUSTER);
+            ebeanAlert.setUuid(uuid);
+            ebeanAlert.setMetric(TEST_METRIC);
+            ebeanAlert.setContext(TEST_CONTEXT);
+            ebeanAlert.setService(TEST_SERVICE);
+            Ebean.save(ebeanAlert);
+            transaction.commit();
+        } finally {
+            transaction.end();
+        }
+
+        final Optional<Alert> expected = alertRepo.get(uuid);
+        Assert.assertTrue(expected.isPresent());
+        Assert.assertEquals(uuid, expected.get().getId());
+        Assert.assertEquals(TEST_CLUSTER, expected.get().getCluster());
+        Assert.assertEquals(TEST_METRIC, expected.get().getMetric());
+        Assert.assertEquals(TEST_EXTENSIONS, expected.get().getExtensions());
+        Assert.assertEquals(TEST_SERVICE, expected.get().getService());
+        Assert.assertEquals(TEST_NAME, expected.get().getName());
+        Assert.assertEquals(TEST_OPERATOR, expected.get().getOperator());
+        Assert.assertEquals(Period.seconds(TEST_PERIOD_IN_SECONDS), expected.get().getPeriod());
+        Assert.assertEquals(TEST_STATISTIC, expected.get().getStatistic());
+        Assert.assertEquals(TEST_QUANTITY, expected.get().getValue());
+        Assert.assertEquals(TEST_CONTEXT, expected.get().getContext());
```

Comment:
So you should probably use instances instead of per field constants and have an externalized equality method. External because if you write an equals on the bean then that will be used by the orm and I am not sure we want to go there (at least not yet).